### PR TITLE
Updated Dates for PARADA and SPKIR cal sheets

### PR DESCRIPTION
**Pull request by funnyabc**
Update dates to reflect date of calibration and not deployment dates. Updtaed spkir lookup table to accurate asset_uid, SN pairing and put in condition to ignore .DS_Store files when using scripts.

### CODE REVIEW

File: tools/cal_scripts/ctd_cal_parser.py [hunk 1]
```diff
@@ -151,6 +152,8 @@ def main():
     lookup = get_uid_serial_mapping('CTD/ctd_lookup.csv')
     for path, directories, files in os.walk('CTD/manufacturer'):
         for file in files:
+            if file == '.DS_Store':
```
Comment (danmergens) [inline tools/cal_scripts/ctd_cal_parser.py]: if you're skipping this hidden file, might as well skip all hidden files: `if file[0] == '.':`

File: tools/cal_scripts/ctd_cal_parser.py [hunk 2]
```diff
@@ -109,6 +109,7 @@ def read_cal(self, filename):
                     self.serial = '16-'
 
                 if key == 'SERIALNO':
+                    print('sn found')
```
Comment (danmergens) [inline tools/cal_scripts/ctd_cal_parser.py]: either use logging service or remove if it's just a temporary statement.

Comment (funnyabc) [reply tools/cal_scripts/ctd_cal_parser.py]: This was a debugging statement and will be removed in the next commit

File: tools/cal_scripts/ctd_cal_parser.py (hunk 1 above)
Comment (funnyabc) [reply tools/cal_scripts/ctd_cal_parser.py]: See latest commit

**Pull request merged**